# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-dreamcast-maple.patch.superh
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.15.1/include/linux/maple.h.vanilla	2025-06-04 14:46:27.000000000 +0200
+++ linux-6.15.1/include/linux/maple.h	2025-06-10 00:31:02.500775517 +0200
@@ -3,6 +3,7 @@
 #define __LINUX_MAPLE_H
 
 #include <mach/maple.h>
+#include <linux/device.h>
 
 struct device;
 
